TestRailAutomationTest/Page/LoginPage.cs

```diff
@@ -0,0 +1,43 @@
+using OpenQA.Selenium;
+using TestRailAutomationTest.Model;
+using TestRailAutomationTest.Service;
+using TestRailAutomationTest.Utils;
+
+namespace TestRailAutomationTest.Page;
+
+public class LoginPage : BasePage
+{
+    private readonly string _pageUrl = DataReader.GetConfig().AppUrl;
+    private readonly By _emailInputLocation = By.XPath("//*[@id=\"name\"]");
+    private readonly By _passwordInputLocation = By.XPath("//*[@id=\"password\"]");
+    private readonly By _searchButtonLocation = By.XPath("//button[contains(@class,\"loginpage-button\")]");
+    
+    public LoginPage(IWebDriver driver) : base(driver)
+    {
+    }
+
+    public LoginPage OpenPage()
+    {
+        Driver.Navigate().GoToUrl(_pageUrl);
```

Comment:
Lets do it in BaseTest because in almost all cases we should open login page 
As I remember it can be done with `Driver.Url = <Url>`